genie-client/src/main/java/com/netflix/genie/client/JobClient.java

```diff
@@ -305,7 +314,9 @@ public void writeTo(final BufferedSink sink) throws IOException {
         @Nullable final Long minFinished,
         @Nullable final Long maxFinished,
         @Nullable final String grouping,
-        @Nullable final String groupingInstance
+        @Nullable final String groupingInstance,
+        @Nullable final Integer page,
+        @Nullable final Integer size
```

Comment:
This is an API breaking change. Anyone using `getJobs` today will get compilation error after updating.
Can you instead create a new method and telescope?